errors.go

```diff
@@ -0,0 +1,5 @@
+package relayer
+
+import "errors"
+
+var ErrDupEvent = errors.New("duplicate: event already exists")
```

Comment:
maybe move this to `relayer/storage` package? so it's easier to find for a future `relayer/storage/another_db_backend` implementation. otherwise, it's also a bit unclear in which context a "duplicate event" error may appear.